github/ci/services/prometheus-stack/patches/stack/production-performance-workloads/kube-prometheus-stack.yaml

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
I think that for the patch not all the fields are required, but should be ok too if they are present

Reply:
Humm, that is right, we don't need the fields that are not changing, but it might not be a problem to keep them